examples/webhooks/tsconfig.json

```diff
@@ -12,10 +12,11 @@
     "baseUrl": "./",
     "incremental": true,
     "skipLibCheck": true,
-    "strictNullChecks": false,
+    "strictNullChecks": true,
```

Comment:
Nice 👌

Reply:
Unfortunately, I'll removed it from this PR due to the requested changes above